### DISCUSSION TITLE
Welcome to CoRE4INET Discussions!

### DISCUSSION THREAD

**Discussion by Musteblume** [CoRE-RG/CoRE4INET#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Lchen429** [comment]
I have a question!
Why extends IOutputQueue  signed “no such component: 'IOutputQueue'”；
All file extends IOutputQueue  have this error。
And other error。
![image](https://github.com/user-attachments/assets/fc87203a-5d29-475c-bf74-8f684b77fc8c)
Thanks!

**Reply by Lchen429** [reply]
Environment ： windows 10，omnet++ 6.1
